src/components/SelectionList/BaseSelectionList.js

```diff
@@ -81,10 +115,17 @@ function BaseSelectionListRadio(props) {
         // because React Native accounts for it in getItemLayout
         itemLayouts.push({length: 0, offset});
 
+        if (selectedCount > 1 && !canSelectMultiple) {
+            Log.alert(
+                'Dev error: SelectionList - multiple items are selected but prop `canSelectMultiple` is false. Please enable `canSelectMultiple` or make your list have only 1 item with `isSelected: true`.',
+            );
+        }
```

Comment:
Why do we need this?

Reply:
Because the list items are controlled by the pages itself, and that's a warning to prevent the developer from misusing the props. If `canSelectMultiple` is false, we expect that inside the items, we will have at max 1 with `isSelected: true`.

```jsx
<SelectionList
  sections={[
    data: [
      {text: 'Item 1', isSelected: true, ...},
      {text: 'Item 2', isSelected: false, ...},
      ...
    ]
  ]}
/>
```